src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Backup/BackupAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -26,12 +26,16 @@
 namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
 {
     /// <summary>
-    /// Backup Item
+    /// Enables backup of an item protected by the recovery services vault.
+    /// Returns the corresponding job created in the service to track this backup operation.
     /// </summary>
     [Cmdlet(VerbsData.Backup, "AzureRmRecoveryServicesBackupItem"), OutputType(typeof(JobBase))]
     public class BackupAzureRmRecoveryServicesBackupItem : RecoveryServicesBackupCmdletBase
     {
-        [Parameter(Mandatory = true, HelpMessage = ParamHelpMsgs.Item.ProtectedItem, 
+        /// <summary>
+        /// The protected item on which backup has to be enabled.
```

Comment:
The term enabled is misleading. This commandlet takes adhoc backup. Can you rephrase it?